asd-aituber/apps/web/__tests__/utils/vrm-loader.hotfix.test.ts

```
@@ -0,0 +1,131 @@
+  it('should use AbortController fallback when timeout not supported', async () => {
+    // Mock successful response
+    vi.mocked(fetch).mockResolvedValueOnce({
+      ok: true,
+      status: 200
+    } as Response)
+
+    // timeoutメソッドを削除してテスト
+    const original = (AbortSignal as any).timeout
+    delete (AbortSignal as any).timeout
+    
+    const result = await checkVRMFileExists('/models/test.vrm')
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)